pkgs/applications/science/logic/kissat/default.nix

```
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, drat-trim }:
+
+stdenv.mkDerivation rec {
+  pname = "kissat";
+  version = "2.0.1-abfa45f";
```

Will `2.0.1` always point to this commit? If yes, maybe:
```suggestion
  version = "2.0.1";
```

(I cannot make my mind if https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming allows doing `2.0.1-abfa45f` or not)

---

There's no official (or tag) release yet. I can't guarantee but I think this commit is the only one for 2.0.1. So I'd like to take 'kissat-2.0.1'.